src/services/review/ReviewRequestService.ts

```diff
@@ -7,8 +7,8 @@ import { ReviewMeta } from "@database/models/ReviewMeta"
 import { ReviewRequest } from "@database/models/ReviewRequest"
 import { Site } from "@root/database/models/Site"
 import { User } from "@root/database/models/User"
-import { RawFileChangeInfo } from "@root/types/github"
-import { FileChangeInfo } from "@root/types/review"
+import { EditedItemDto, FileType } from "@root/types/dto /review"
```

Comment:
err i'd prefer not to - the dto is an api contract between teh FE/BE. The fact that this is functionally the same as `EditedItemProps` shouldn't matter because the props can be free to change but when it hits `apiService` should still be `EditedItemDto` to ensure that stuff is alw compatible b/w FE/BE